test/pthread/main_js_with_loader.html

```diff
@@ -147,28 +147,8 @@
       };
     </script>
 
-    <script>
-
-function download(url) {
-  return fetch(url).then((rsp) => rsp.blob());
-}
-
-function addToDom(scriptCodeBlob) {
-  return new Promise((resolve, reject) => {
-    var script = document.createElement('script');
-    var objectUrl = URL.createObjectURL(scriptCodeBlob);
-    script.src = objectUrl;
-    script.onload = () => {
-      Module['mainScriptUrlOrBlob'] = scriptCodeBlob;
-      URL.revokeObjectURL(objectUrl);
-      resolve();
-    }
-    document.body.appendChild(script);
-  });
-}
-
-download('hello_thread_with_blob_url.js').then(addToDom);
-
-    </script>
+    <!-- Exactly one of these two scripts will exist -->
+    <script src="loader.js"></script>
+    <script type="module" src="loader.mjs"></script>
```

Comment:
Actually I wonder if that loader can always be an `mjs` file ..even when its loading a plain `js` file?   (i.e. can we remove that first of these two lines?

Reply:
Good idea!